docs/conf.py

```diff
@@ -91,7 +91,7 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'flakeheaven', 'FlakeHeaven Documentation',
+    (main_doc, 'flakeheaven', 'FlakeHeaven Documentation',
```

Comment:
```suggestion
    (root_doc, 'flakeheaven', 'FlakeHeaven Documentation',
```

last one, https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-master_doc